tests/common/Configuration.cs

```diff
@@ -617,6 +623,8 @@ public static string GetBaseLibrary (Profile profile)
 			switch (profile) {
 			case Profile.iOS:
 				return XamarinIOSDll;
+			case Profile.macOSSystem: // MacCatalyst
+				return XamarinCatalystDll;
```

Comment:
This doesn't look right, and I think with the `Configuration.GetRefLibraries ()` code from above this shouldn't be needed.